.eslintignore

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = false
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{json,yml,md,babelrc,eslintrc,remarkrc}]
+indent_style = space
+indent_size = 2
```

Comment:
can we make the indent size 4 spaces? i think this is what we have in ql, shouldn't we keep the same formatting rules for js too?

Reply:
done